test/org/sagebionetworks/bridge/play/controllers/AuthenticationControllerTest.java

```diff
@@ -103,6 +103,9 @@ public void testCode() throws Exception {
 
                 String output = jedisOps.get(sessionToken);
                 assertNull("Should no longer be session data", output);
+                } catch(Exception e) {
+                    e.printStackTrace();
+                }
```

Comment:
As per IM conversation, Play does in fact produce a stack trace, so we should remove the catch block.